src/pages/styles/icons.tsx

```diff
@@ -0,0 +1,128 @@
+  allIcons,
+  IconKeys,
+  allStaticIcons,
+  StaticIconKeys,
+  SuomifiIcon,
+} from 'suomifi-icons';
+
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import Annotation from 'components/Annotation';
+import ComponentDescription from 'components/ComponentDescription';
+import sideNavData from 'config/sidenav/styles';
+import ComponentExample from 'components/ComponentExample';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+import { ReactComponent as Slack } from 'icons/slack.svg';
+
+const iconCategories = [
+  {
+    id: 'baseIcons',
+    icons: allIcons,
+  },
+  {
+    id: 'illustrativeIcons',
+    icons: allStaticIcons.filter(icon => icon !== 'toggle'),
+  },
+];
+const StyleIcon = styled(SuomifiIcon)`
+  height: 1.5em;
+  width: 1.5em;
+`;
+
+const getExampleIcon = (
+  id: string,
+  label: string,
+  style?: CSSProperties,
+): JSX.Element => (
+  <div
+    key={id}
+    style={{
+      margin: `${suomifiTheme.spacing.s} ${suomifiTheme.spacing.xl} ${
+        suomifiTheme.spacing.l
+      } 0`,
+      lineHeight: '1rem',
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'space-around',
+      alignItems: 'center',
+      width: '6rem',
+      height: '6rem',
+      ...style,
+    }}
+  >
+    <StyleIcon icon={(id as any) as IconKeys | StaticIconKeys} />
+    <div>{label}</div>
+  </div>
+);
+
+const Page = (): JSX.Element => (
+  <NamespacesConsumer ns={['icons']}>
+    {t => (
+      <Layout sideNavData={sideNavData(t)}>
+        <SEO title={t('title')} />
+        <Heading.h1>{t('title')}</Heading.h1>
+
+        <Paragraph.lead>
+          <Text.lead>{t('intro')}</Text.lead>
+        </Paragraph.lead>
+
+        {iconCategories.map(item => (
+          <ComponentDescription
+            key={item.id}
+            mainTitle={t(`${item.id}.title`)}
+            description=""
+            exampleFirst={false}
+            noCode
+          >
+            <ComponentExample
+              style={{
+                padding: 0,
+                justifyContent: 'flex-start',
+                background: 'none',
+              }}
+            >
+              {item.icons.map(icon => {
+                return item.id === 'baseIcons'
+                  ? getExampleIcon(icon, t(`${icon}.label`))
+                  : getExampleIcon(icon, t(`${icon}.label`), {
+                      width: '256px',
+                    });
+              })}
+            </ComponentExample>
+          </ComponentDescription>
+        ))}
+        <Annotation
+          title={t('note.title')}
+          description={t('note.description')}
+          link={{
+            icon: (
+              <Slack
+                style={{
+                  fill: suomifiTheme.colors.whiteBase,
+                  fontSize: '16px',
+                }}
+              />
+            ),
+            text: t('note.link.text'),
+            url: t('note.link.url'),
+          }}
+        />
```

Comment:
Is it necessary to tell this separately for icons usage? 
As other pages doesn't have this kind of annotation and there is same Slack-link on the page's footer?

Reply:
This is now implemented according to designs, but I do like the idea of keeping things uniform rather than adding this to only Icons page. Will take this question to designers and continue this thread a separate issue in case needed.